ql/src/java/org/apache/hadoop/hive/ql/exec/tez/TezAvailableSlotsCalculator.java

```diff
@@ -39,6 +43,9 @@ public int getAvailableSlots() {
         }
         int totalResource = inputInitializerContext.getTotalAvailableResource().getMemory();
         int taskResource = inputInitializerContext.getVertexTaskResource().getMemory();
-        return totalResource / taskResource;
+        int availableSlots = totalResource / taskResource;;
+        LOG.debug("totalResource: {} / taskResource: {} =  availableSlots: {}", totalResource, taskResource,
```

Comment:
could we please add the metric "Mb" since we output the resource information?

Reply:
makes sense! thanks, I'll do

Reply:
done